TestsExample/src/Test865.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { View, Text, Button } from 'react-native';
+
+import { NavigationContainer } from '@react-navigation/native';
+import { createNativeStackNavigator } from 'react-native-screens/native-stack';
+
+const Stack = createNativeStackNavigator();
+
+const First = ({ navigation }: any) => (
```

Comment:
Please use proper `navigation` type, you can take it from here: https://github.com/software-mansion/react-native-screens/blob/master/TestsExample/src/Test817.tsx#L35. Same for screens below.

Reply:
Fixed in c752f4ec74603d581e8657bf0684436f23ac2041